components/SpotDetails/SpotSocial/SpotSocial.js

```diff
@@ -0,0 +1,29 @@
+import SpotSocialComment from "./SpotSocialComment";
+import SpotSocialForm from "./SpotSocialForm";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function SpotSocial({ ID }) {
+  const [comments, setComments] = useState([]);
+  console.log(comments);
```

Comment:
Please delete this `console.log`. In your final PR should be no console logs anymore.